src/N98/Magento/Command/LocalConfig/GenerateCommand.php

```diff
@@ -41,52 +41,52 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->detectMagento($output);
-        $configFile = $this->_getLocalConfigFilename();
+        $configFile = $this->_magentoRootFolder . '/app/etc/local.xml';
         $configFileTemplate = $this->_magentoRootFolder . '/app/etc/local.xml.template';
-        if (!file_exists($configFile)) {
-            $this->writeSection($output, 'Generate Magento local.xml');
-            $this->askForArguments($input, $output);
-            if (!file_exists($configFileTemplate)) {
-                $output->writeln('<error>File ' . $this->_magentoRootFolder . '/app/etc/local.xml.template does not exist.</error>');
-                return;
-            }
 
-            if (!is_writable($this->_magentoRootFolder . '/app/etc')) {
-                $output->writeln('<error>Folder ' . $this->_magentoRootFolder . '/app/etc is not writeable</error>');
-                return;
-            }
+        if (file_exists($configFile)) {
+            $output->writeln(sprintf('<info>local.xml file already exists in folder "%s/app/etc"</info>', $this->_magentoRootFolder));
+            return;
+        }
 
-            $content = file_get_contents($configFileTemplate);
-            $key = $input->getArgument('encryption-key') ? $input->getArgument('encryption-key') : md5(uniqid());
-            if (is_array($key)) {
```

Comment:
@AydinHassan see my comment here https://github.com/fooman/n98-magerun/commit/e804d7d3ad8d1f4e1e6b211518651eaafed6dcb0#comments - I never worked out why, at times, the getArgument would return an array. Adding the extra check was quicker than trying to replicate what versions and circumstances brought it up. Since it looks like you are adding test coverage, happy to see it go.